mlir/include/mlir/Dialect/LLVMIR/LLVMTypes.h

```diff
@@ -226,8 +87,9 @@ void printType(Type type, AsmPrinter &printer);
 
 /// Parse any MLIR type or a concise syntax for LLVM types.
 ParseResult parsePrettyLLVMType(AsmParser &p, Type &type);
+ParseResult parsePrettyLLVMType(AsmParser &p, SmallVectorImpl<Type> &types);
 /// Print any MLIR type or a concise syntax for LLVM types.
-void printPrettyLLVMType(AsmPrinter &p, Type type);
+void printPrettyLLVMType(AsmPrinter &p, TypeRange type);
```

Comment:
```suggestion
void printPrettyLLVMType(AsmPrinter &p, TypeRange types);
```
ultra nit: It could make sense to rename to printLLVMTypeList and parseLLVMTypeList above?